docs/ros/ros/installation/robot.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Robot installation
+sidebar_label: Robot
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+### Operating System (OS) {#operating-system}
+
+ROS 2 Humble uses [Ubuntu 22.04](https://releases.ubuntu.com/jammy/) as its Tier 1 operating system. Though other operating systems are supported,
+it is highly recommended to use Ubuntu 22.04. For the robot PC, Ubuntu 22.04 Server should be installed. The server version of the OS will not install a
+graphical user interface (GUI).
+
+### Networking
+
+Once the operating system is installed, the robot PC needs to be connected to the internet. TODO.
```

Comment:
Noting that there is still a "TODO" here.

Reply:
Not sure if we want networking here or under the robot section.

Reply:
You can possibly refer them back to the robot if there are any robot-specific details, but you can probably use something here similar to what is currently in the tutorials_husky_ros1.mdx and making it slightly more generic:

### Setting up Husky's Network Configuration {#husky-networking}

Husky is normally equipped with a combination Wi-Fi + Bluetooth module.
If this is your first unboxing, ensure that Husky's wireless antennae are firmly screwed on to the chassis.
Some Husky robots may only be equipped with a single antenna, depending on the exact model of PC installed in the robot.

#### First Connection

By default, Husky's Wi-Fi is in client mode, looking for the wireless network at the Clearpath factory.

<ComponentWiredRobotConnection />

#### Changing the Default Password

<ComponentChangingDefaultPassword />

#### Wi-Fi Setup

<ComponentWifiRobotConnection />

Reply:
The instructions will be different because we will be using the computer setup scripts I believe.

Reply:
Ok, understood.